mne/io/kit/tests/test_kit.py

```diff
@@ -185,4 +188,24 @@ def test_hsp_elp():
     assert_array_almost_equal(pts_elp_in_dev, pts_txt_in_dev, decimal=5)
 
 
+def test_decimate():
+    """Test decimation of digitizer headshapes with too many points. """
+    # load headshape and convert to meters
+    hsp = np.loadtxt(sphere_hsp_txt_path)
+    hsp /= 1000
```

Comment:
It might be better to generate the sphere head shape file during the test (as you did before), and save it in a temporary director before loading again? Including it as a text file in the repository adds 765 KB of data. I am not sure at what size data files for tests become a problem but seem to remember that the issue was raised at some point (@Eric89GXL ?).